mirrord/layer/src/file/hooks.rs

```diff
@@ -54,7 +54,11 @@ type stat64 = stat;
 
 /// Take the original raw c_char pointer and a resulting bypass, and either the original pointer or
```

Comment:
I assume there's a word missing here?
```suggestion
/// Take the original raw c_char pointer and a resulting bypass, and return either the original pointer or
```